scripts/stackstorm.js

```diff
@@ -302,7 +304,9 @@ module.exports = function(robot) {
           data = e.data;
         }
 
-        postDataHandler.postData(data);
+        if (data && data.message) {
```

Comment:
> nothing happened

That's exactly the point: if an empty string is passed, then nothing happened. Until we can use Mistral to avoid `post_message` execution in the workflow or rework announcement runner, that's the most we can do.